src/Community/Who-Is-Using-Orleans.md

```diff
@@ -48,6 +48,9 @@ Argentinian-based electronic brokerage firm
 Nomnio IoT Platform and industry projects where reliability and performance is required
 * <a href="https://www.realartists.com/index.html" target="_blank">__Real Artists Ship 2.0__</a>
 Fast, native, comprehensive issue tracking for GitHub 
+* <a href="https://mri-inc.net/" target="_blank">__Manufacturing Resources International__</a>
```

Comment:
odd, I wonder why these are using html tags and not just markdown for the links. Just a comment for @sergeybykov . Nothing for you to do @DarkCow as this is consistent with the rest.